community/neo4j/src/docs/ops/upgrades.asciidoc

```diff
@@ -1,21 +1,25 @@
 [[deployment-upgrading]]
-Upgrading
-=========
+= Upgrading
+
+:manual-base-url: http://neo4j.com/docs/{neo4j-version}
+:manual-ha-upgrade-guide: {manual-base-url}/ha-upgrade-guide.html
+:manual-cypher-compatibility: {manual-base-url}/cypher-compatibility.html
 
 [IMPORTANT]
 This section describes upgrading a single Neo4j instance.
-Upgrading a Neo4j HA cluster (Neo4j Enterprise) requires a very specific process be followed.
-Please see 
+Upgrading a Neo4j HA cluster (Neo4j Enterprise) requires a very specific procedure be followed.
```

Comment:
How about "To upgrade ... a very specific procedure must be followed"? Otherwise it should be either "requires a very specific procedure **to** be followed" or "requires **that** a very specific procedure be followed".